RELEASING.md

```diff
@@ -193,21 +195,40 @@ at the same time that subcomponent release branches are cut, often well before t
    ```
    git checkout -b release-candidate-vX.Y
    ```
+   
+1. Enable branch deployments for this branch on netlify by adding branch name to `Deploy contexts` in site settings.
 
-1. In [netlify.toml](netlify.toml), set the `CANDIDATE_RELEASE` environment variable:
-
-   ```toml
-   [build.environment]
-     CANDIDATE_RELEASE = "vX.Y"
-   ```
+1. Add a new stanza for the build of this branch in [netlify.toml](netlify.toml). 
 
+    ```toml
+   [build.<branch-name>]
+      make bin/helm
+      export PATH=$PATH:$(pwd)/bin
+      jekyll build --config _config.yml,$(pwd)/netlify/_config_noindex.yml --baseurl /vX.Y
+      jekyll build --config _config.yml,$(pwd)/netlify/_config_noindex.yml,$(pwd)/netlify/_manifests_only.yml --baseurl /vX.Y --destination _site/vX.Y
+    ```
+   
 1. Commit your changes. For example:
 
    ```
    git commit -m "build vX.Y candidate"
    ```
+   
+1. Open a pull request to the upstream release-candidate-vX.Y branch. Get it reviewed and merged. 
 
-1. Push your branch and open a pull request to the upstream master branch. Get it reviewed and wait for it to pass CI.
+1. After ensuring that the branch deployment is finished, in production branch's [netlify.toml](netlify.toml), add a new stanza for proxy to branch deploy site of release candidate.
+
+   ```toml
+   [[redirects]]
+     from = "/vX.Y/*"
+     to = "https://<branch-name>--calico-test.netlify.app/:splat"
```

Comment:
lets use `release-vX.Y` instead of `<branch-name>`.

And lets use `calico` instead of `calico-test`.

```suggestion
     to = "https://release-vX.Y--calico.netlify.app/:splat"
```